corehq/apps/products/models.py

```diff
@@ -297,7 +297,7 @@ class SQLProduct(models.Model):
     name = models.CharField(max_length=100, null=True)
     is_archived = models.BooleanField(default=False)
     code = models.CharField(max_length=100, default='', null=True)
-    description = models.CharField(max_length=100, null=True, default='')
+    description = models.TextField(null=True, default='')
```

Comment:
we might also get away with the `null=True` with when default is set to empty string. not sure would have to check the db